WISE-Community/WISE-Client
# refactor(Classroom Monitor): Delete Dashboard Controller

## Changes

Deleted Dashboard Controller.

## Test

- Make sure the Classroom Monitor loads properly
- Make sure the dashboard route no longer works `http://localhost:81/teacher/manage/unit/<unit number>/dashboard`

Closes #974
